test-runner-frontend/src/Components/TestRunsTable.js

```diff
@@ -0,0 +1,241 @@
+import * as React from 'react';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import HighlightOffOutlinedIcon from '@mui/icons-material/HighlightOffOutlined';
+import CheckCircleOutlinedIcon from '@mui/icons-material/CheckCircleOutlined';
+import Checkbox from '@mui/material/Checkbox';
+import { IconButton, Stack, Typography } from '@mui/material';
+import ProgressBar from 'react-bootstrap/ProgressBar';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+
+/*
+const columns = [
+  { id: 'name', label: 'Name', minWidth: 170 },
+  { id: 'code', label: 'ISO\u00a0Code', minWidth: 100 },
+  {
+    id: 'population',
+    label: 'Population',
+    minWidth: 170,
+    align: 'right',
+    format: (value) => value.toLocaleString('en-US'),
+  },
+  {
+    id: 'size',
+    label: 'Size\u00a0(km\u00b2)',
+    minWidth: 170,
+    align: 'right',
+    format: (value) => value.toLocaleString('en-US'),
+  },
+  {
+    id: 'density',
+    label: 'Density',
+    minWidth: 170,
+    align: 'right',
+    format: (value) => value.toFixed(2),
+  },
+];
+*/
+
+
+const columns = [
+
+    { 
+        id: 'id',
+        label: 'JobId',
+        minWidth: 330,
+
+    },
+    {
+      id: 'branch',
+      label: 'Branch',
+      minWidth: 150,
+    },
+    {
+      id: 'benchmarks',
+      label: 'Benchmarks',
+      minWidth: 180,
+    },
+    {
+      id: 'improvedVsRegresed',
+      label: 'Improved vs Regresed',
+      type: 'number',
+      minWidth: 250,
+    },
+    {
+      id: 'date',
+      label: 'Date',
+      description: 'This column has a value getter and is not sortable.',
+      minWidth: 300,
+    },
+  ];
+
+function createData(name, code, population, size) {
+  const density = population / size;
+  return { name, code, population, size, density };
+}
+
+export default function TestRunsTable({childToParentCheckBox, checkBoxState, testResults, rowsF, sortBenchmarks, sortDate}) {
```

Comment:
its a little unclear with what some of these props are. this page actually has good recommendations for prop namings that are very common for react users - https://dlinau.wordpress.com/2016/02/22/how-to-name-props-for-react-components/. can we look at renaming these variables to fit this pattern?